README.md

```diff
@@ -39,143 +40,9 @@ make bootstrap && make compile-test
-the store will Call `GenerateStats()`on all of its stat generators, and flush all the Counters and Gauges registered with it.
-
-### Scopes
-
-Statistics can be namespaced by creating a Scope:
-
-```Go
-store := stats.NewDefaultStore()
-scope := stats.Scope("service")
-// the following counter will be emitted at the stats tree rooted at `service`.
-c := scope.NewCounter("success")
-```
-
-Additionally you can create subscopes:
-
-```Go
-store := stats.NewDefaultStore()
-scope := stats.Scope("service")
-networkScope := scope.Scope("network")
-// the following counter will be emitted at the stats tree rooted at `service.network`.
-c := networkScope.NewCounter("requests")
-```
-
-### Types of Stats
-
-#### Counters
-
-Counters are an always incrementing stat. Counters implement the following interface:
-
-```Go
-type Counter interface {
-	Add(uint64)     // increment the counter by the argument's value.
-	Inc()           // increment the counter by 1.
-	Set(uint64)     // sets an internal counter value which will be written in the next flush. Its use is discouraged as it may break the counter's "always incrementing" semantics.
-	String() string // return the current value of the counter as a string.
-	Value() uint64  // return the current value of the counter as a uint64.
-}
-```
-
-Counters are added to a store, or a scope by using the `NewCounter(name string)` method.
-
-#### Gauges
-
-A Gauge can both increment and decrement. Gauges implement the following interface:
-
-```Go
-type Gauge interface {
-	Add(uint64)      // increment the gauge by the argument's value.
-	Sub(uint64)      // decrement the gauge by the argument's value .
-	Inc()            // increment the gauge by 1.
-	Dec()            // decrement the gauge by 1.
-	Set(uint64)      // set the gauge to the argument's value.
-	String() string  // return the current value of the gauge as a string.
-	Value() uint64   // return the current value of the gauge as a uint64.
-}
-```
-
-Gauges are added to a store, or a scope by using the `NewCounter(name string)` method.
-
-#### Timers
-
-Timers can be used to flush timing statistics. Timers implement the following interface:
-
-```Go
-type Timer interface {
-	AddValue(float64)        // flushes the timer with the argument's value.
-	AllocateSpan() Timespan  // allocates a Timespan.
-}
-```
-
-Timers are added to a store, or a scope by using the `NewTimer(name string)` method.
-
-Timespans can be used to measure spans of time. They measure time from the time they are allocated by a Timer with `AllocateSpan()`,
-until they call `Complete()`. When `Complete()` is called the timespan is flushed. A Timespan can be flushed at function
-return by calling `Complete()` with golang's `defer` statement.
-
-### Stats with Tags
-
-Counters, Gauges and Timers can have tags in addition to their names. All three have methods 
-`New[Counter|Gauge|Timer]WithTags(name string, tags map[string]string)`.
```

Comment:
I'd recommend an example usage of some of these stats in the readme (as well as the godocs, which can be a different PR when you have the opportunity).